docs/development/telemetry.md

```diff
@@ -0,0 +1,11 @@
+Open Telemetry is all set up to provide data to a visualizer.
+During development, we set up SigNoz to show data.
+Install it by:
+
+```shell
+git clone -b main https://github.com/SigNoz/signoz.git
+cd signoz/deploy/
+./install.sh
+```
+
+Then go to http://localhost:3301/services/ZAC and see the data coming in.
```

Comment:
I don't see that port 3301 is exposed in our Dockerfile?

Reply:
Even nicer would be to add an integration test to our integration test suite to check if this endpoint is accessible and gives back the expected data. Should be quite easy to add?

Reply:
The acutal vizualization software I used (signoz) is not included. You install it by doing the steps in this file. It runs locally and exposes 3301

Reply:
I'm not talking about visualization. I'm talking about adding an integration test to make sure that the OpenTelemetry endpoint in the ZAC Docker container is accessible and works as expected. I think an integration test is perfect for that.